docs/index.rst

```diff
@@ -12,6 +12,17 @@ NI-XNET Python Documentation
    :maxdepth: 3
    :caption: API Reference:
 
+   session
+   base
+   frames
+   signals
+   types
+   errors
+   constants
+   collection
+   intf
+   j1939
```

Comment:
We have references to things in 'base.py' for which Sphinx complains if an RST file is created but not added to a TOCtree. Maybe there's a way to have it be generat doc content without adding to a TOC?

We need to revamp 'index.rst' anyway (because it's only showing API reference in the TOC section) so we can fix this then.

We also need to cleanup the code for relying documentaion in '_enums.py', i.e. enums.rst.
Ideally that documentation would go in a public module....maybe 'constants.py' but I didn't have the time to do that work.